app/views/projects/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2> Project name: <%=@project.name %></h2>
+  <p>Material: <%=@project.material %></p>
+  <p>Challenge Theme: <%= @theme %></p>
+  <p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
For number of contestants, we don't want to do math logic here. We would want to do that in the model. So I would create a method in your `Projects` model
```ruby
def num_contestants
  contestants.count
end
``` 
then call that method in here - `@project.num_contestants`